nntrainer/layers/bn_layer.cpp

```diff
@@ -89,6 +110,18 @@ void BatchNormalizationLayer::finalize(InitLayerContext &context) {
   wt_idx[BNParams::deviation] =
     context.requestTensor(in_dim, context.getName() + ":deviation",
                           Tensor::Initializer::NONE, false, ITERATION_LIFESPAN);
+  wt_idx[BNParams::invstd] =
+    context.requestTensor(dim, context.getName() + ":invstd",
+                          Tensor::Initializer::NONE, false, ITERATION_LIFESPAN);
+  wt_idx[BNParams::t_reduced] =
+    context.requestTensor(dim, context.getName() + ":tensor_reduced",
+                          Tensor::Initializer::NONE, false, ITERATION_LIFESPAN);
+  wt_idx[BNParams::t_full_fw] = context.requestTensor(
+    in_dim, context.getName() + ":tensor_full_fw", Tensor::Initializer::NONE,
+    false, FORWARD_FUNC_LIFESPAN);
+  wt_idx[BNParams::t_full_bw] = context.requestTensor(
```

Comment:
Just for checking, if we set FORWARD_FUNC_LIFESPAN here, is it null in backwarding?

Reply:
currently, it is in available backward (run context doesn't know when function is calling it), but later we will have a check so `context.requestTensor()` will throw in this case.

Reply:
I have added a corresponding todo in layer context.